projects/saturn/vocabulary.ttl

```diff
@@ -211,31 +199,15 @@ sils:Project a rdfs:Class, sh:NodeShape ;
         sh:datatype xsd:string ;
         sh:maxCount 1 ;
         sh:path sils:projectDescription
-    ],
-    [
-        sh:name "Principal Investigator" ;
-        sh:description "The investigator this study belongs to." ;
-        sh:maxCount 1 ;
-        sh:class sils:PrincipalInvestigator ;
-        sh:path sils:principalInvestigator
-    ],
-    [
-        sh:name "Studies" ;
-        sh:description "Linked studies" ;
-        sh:path [sh:inversePath sils:project]
-    ],
-    [
-        sh:name "Entity location" ;
-        sh:description "Location of the entity in the browser." ;
-        sh:class fs:Directory ;
-        sh:path [sh:inversePath fs:linkedEntity]
     ] .
 
 sils:studyIdentifier a rdf:Property .
 sils:studyTitle a rdf:Property .
 sils:studyDescription a rdf:Property .
-sils:principalInvestigator a rdf:Property .
-sils:project a rdf:Property .
+sils:studyGoal a rdf:Property .
+sils:studyOrganisms a rdf:Property .
```

Comment:
do we need the 'study' prefixes? I would say there are 'organisms' and a study can link to it, not 'studyOrganisms'

Reply:
the prefix is there to make sure this a property of a study entity. The class could be without prefix, but for some properties it's needed, e.g. if a sample would have to be linked to organisms, it's property would have to be sampleOrganisms